src/Lykke.Service.ExchangeConnector/Handlers/TradingSignalsHandler.cs

```diff
@@ -113,9 +114,9 @@ public override Task Handle(TradingSignal message)
 
                         try
                         {
-                            bool orderCanceled = await exchange.CancelOrder(signal, translatedSignal);
-
-                            if (orderCanceled)
+                            var executedTrade = await exchange.CancelOrderAndWaitExecution(signal, translatedSignal, apiTimeout);
```

Comment:
Let's call ExectuedTradeHandlers with this already constructed ExectuedTrade entity, we don't have to recreate it later.

Reply:
Fixed.